internal/app/ldap/ldap.go

```diff
@@ -0,0 +1,74 @@
+package ldap
+
+import (
+	"fmt"
```

Comment:
let organize import correctly. normally there will be 3 blocks:
- built-in
- external
- internal